go/host/host.go

```diff
@@ -547,8 +561,21 @@ func (h *host) publishRollup(producedRollup common.ExtRollup) {
 
 // Creates a batch based on the rollup and distributes it to all other nodes.
 func (h *host) distributeBatch(producedRollup common.ExtRollup) {
-	// TODO - #718 - Store batch
-	// TODO - #718 - Distribute batch
+	batch := common.ExtBatch{
```

Comment:
I assume this is just the placeholder for future stuff

Reply:
Yeah, it'll evolve over time. Just wanted to plumb it in gradually. First focus is to have rollups match against batches 1-1, but start having the RPC calls reflect the contents of the batches. There's a description of the stages here: https://github.com/obscuronet/obscuro-internal/issues/718.